docs/docs/recipes/openid-connect/authentication-parameters/README.md

```diff
@@ -0,0 +1,85 @@
+import Availability from '@components/Availability';
+
+# Authentication parameters
+
+In addition to the parameters required by the [OpenID Connect Core specification](https://openid.net/specs/openid-connect-core-1_0.html#AuthRequest), Logto supports the following parameters in the authentication request for better customization:
+
+- `first_screen`: The first screen to show for the authentication process.
+- `direct_sign_in`: Whether to skip the first screen and invoke the sign-in process directly.
+
+## First screen
+
+<Availability cloud oss={{ major: 1, minor: 15 }} />
+
+This parameter allows you to customize the first screen that users see when they start the authentication process. The value can be `signIn` or `register`. If omitted, the default value is `signIn`.
+
+Here's a non-normative example of the authentication request with the `first_screen` parameter:
+
+```bash
+curl --location \
+  --request GET 'https://[tenant-id].logto.app/oidc/auth?client_id=1234567890&...&first_screen=register'
+```
+
+:::note
+When `first_screen` is set, the legacy `interaction_mode` parameter is ignored.
+:::
+
+In supported Logto SDKs, you can set the `firstScreen` property when calling the `signIn` method:
+
+```ts title="JavaScript"
+logtoClient.signIn({
+  redirectUri: 'https://your-app.com/callback',
+  firstScreen: 'register',
+});
+```
+
+We are gradually adding support for the `first_screen` parameter to all Logto SDKs. If you don't see it in your SDK, please open an issue or contact us.
```

Comment:
nit: Compared to the previous paragraph, feel like this paragraph should be placed in a note annotation

Reply:
or info?